### PR TITLE
app-vim/lightline: EAPI 6->8, minor fixes, add snapshot version, undo last-rite

### PR DESCRIPTION
**Pull request by NexAdn**
This PR contains a set of patches:

* EAPI is bumped from 6 to 8 (it turned out that no changes were necessary for this)
* a few minor issues with the ebuild were fixed (mainly requirements for snapshot versions)
* a snapshot version was added for the latest master commit
* last-rite is undone as the issues noted in the last-rite are resolved by merging this PR

### CODE REVIEW

**Comment by scardracs** [comment]
If you don't want to l-r please take in consideration to ask to @monsieurp if he is still interested on it and add yourself as co-maintainer (or drop him and add yourself if he isn't interested anymore)

File: app-vim/lightline/lightline-0_pre20221005.ebuild [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit vim-plugin
+
+LIGHTLINE_COMMIT=b1e91b41f5028d65fa3d31a425ff21591d5d957f
```
Comment (arthurzam) [inline app-vim/lightline/lightline-0_pre20221005.ebuild:8]: I prefer that this constant is moved into the `else` block, closer to usage, less places to miss/break.

File: app-vim/lightline/lightline-9999.ebuild [hunk 2]
```diff
@@ -1,21 +1,22 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
 
 inherit vim-plugin
 
+# LIGHTLINE_COMMIT=
```
Comment (arthurzam) [inline app-vim/lightline/lightline-9999.ebuild:8]: I prefer that this constant is moved into the `else` block, closer to usage, less places to miss/break.

**Review by arthurzam — CHANGES_REQUESTED** [review]
Some small changes requested, and I will ping someone from @gentoo/vim

**Comment by arthurzam** [comment]
> If you don't want to l-r please take in consideration to ask to @monsieurp if he is still interested on it and add yourself as co-maintainer (or drop him and add yourself if he isn't interested anymore)

I quite agree. Vim project is very understaffed, and if you are interested in this package, please proxy maintain it. I will still get an approval for this from vim project, so for now add yourself and I will ping them.

**Comment by NexAdn** [comment]
> [...] please proxy maintain it. I will still get an approval for this from vim project, so for now add yourself and I will ping them.

Will do. Do I need to add proxy-maint as project as well or not in this case?

File: app-vim/lightline/lightline-0_pre20221005.ebuild (hunk 1 above)
Comment (NexAdn) [reply app-vim/lightline/lightline-0_pre20221005.ebuild]: done, please check

**Review by arthurzam — APPROVED** [review]
Looks great, thank you for fast fixes!

I've sent a ping to vim project, will see about a reply. I think it would be in near 24h, but maybe faster.

**Review by monsieurp — APPROVED** [review]

**Comment by monsieurp** [comment]
LGTM :) @arthurzam do you want me to merge the PR?

**Pull request closed without merging**